assets/js/touchpoints_translations.js

```diff
@@ -0,0 +1,13 @@
+const yesButton = document.querySelector('input[type=submit][value=yes]');
+const noButton = document.querySelector('input[type=submit][value=no]');
+
+if (document.documentElement.lang === 'es') {
+  if (yesButton) {
+    yesButton.value = 'si';
```

Comment:
Do we have control over the survey answers to make these capitalized, i.e. "Yes", "No", like they are in the designs?

Reply:
I will make it capitalized.